app/views.py

```diff
@@ -0,0 +1,165 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+from app.models import User, Qn, Answer
+
+
+app = Flask(__name__)
```

Comment:
for the documentation, you need import Swagger from flassger and pass in app i.e Swagger(app)